pkgs/development/python-modules/async-upnp-client/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pythonOlder
+, voluptuous, aiohttp, async-timeout, python-didl-lite, defusedxml
+, pytest, pytest-asyncio, pytestcov }:
+
+buildPythonPackage rec {
+  pname = "async-upnp-client";
+  version = "0.14.14";
+  disabled = pythonOlder "3.5";
+
+  src = fetchFromGitHub {
+    owner = "StevenLooman";
+    repo = "async_upnp_client";
+    rev = version;
+    sha256 = "1ysj72l4z78h427ar95x7af0jw0xq1cbca0k8b34vqyyhgs8wc6y";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+    async-timeout
+    defusedxml
+    python-didl-lite
+    voluptuous
+  ];
+
+  checkInputs = [
+    pytest
+    pytest-asyncio
+    pytestcov
+  ];
+
+  checkPhase = ''
+    py.test --cov=async_upnp_client --cov-report=term
```

Comment:
coverage is great to run upstream, but doesn't provide a lot of value for asserting package validity. However, these are cheap to run, so I don't mind including them.

Personally, I wouldn't worry about enabling code coverage for nixpkgs :).

Reply:
Thanks for the feedback regarding cov, I will drop them.